vagrant/tutorial/setup_docker.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+#
+# Small script to start the docker daemon and pull docker images so they are
+# cached for use in the tutorial later.
+
+# Error on any failure of a simple command.
+set -e
+
+# Start or restart the docker daemon.
+systemctl restart docker
+
+# Record the version of docker for the logs.
+echo "Docker Version:"
+docker --version
+
+# Pull docker images so they are cached for later.
+IMAGES="busybox"
+IMAGES+=" clusterhq/mongodb"
+IMAGES+=" redis"
+IMAGES+=" python:2.7-slim"
+IMAGES+=" clusterhq/flask"
```

Comment:
Decided to grep through the bash man page to figure out what was going on. We could have done this, but then below we would need ${IMAGES[*]} ( or "${IMAGES[@]}" might be even better). I'll keep that in mind for next time, although honestly the fact that ${IMAGES} just returns ${IMAGES[0]} makes me sort of not want to use bash Arrays.

For now leaving as quotes given that I've already restarted the tests.